src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -687,6 +707,17 @@ public override void VisitExpressionStatement(ExpressionStatementSyntax node)
 
         public override void VisitLocalDeclarationStatement(LocalDeclarationStatementSyntax node)
         {
+            node.Declaration.Type.VisitRankSpecifiers((rankSpecifier, localBinderFactory) =>
```

Comment:
>node.Declaration.Type.VisitRankSpecifiers((rankSpecifier, localBinderFactory) => [](start = 12, length = 80)

Consider using ```VisitRankSpecifiers``` helper here as well.

Reply:
Like this:
 `VisitRankSpecifiers(node.Declaration.Type, _enclosing);`
?